sdk/py/apepay/manifest.json

```diff
@@ -0,0 +1 @@
+../../../.build/__local__.json
```

Comment:
i am confused by this file, it is not really JSON?

Reply:
This is a soft link back to the build file, so that poetry will bundle it

Reply:
when you install it, it ends up in `site-packages/.build/`

Reply:
I suppose a better solution is to just cp it into the directory when it gets built :thinking:

Reply:
It fails now if you don't build first, so I like this approach as it's more fail-safe